lms/extensions/feature_flags/_helpers.py

```diff
@@ -65,6 +88,9 @@ def set(self, response, payload):
 
     def get(self):
         jwt_bytes = self._request.cookies.get(self._name, "")
+        if not jwt_bytes:
+            return {}
```

Comment:
Separated this out from the code that was catching us putting blank values into the next part. Very confusing saying you have an invalid token, when the actual problem is you don't have one at all.